code/game/objects/items/storage/firstaid.dm

```diff
@@ -378,13 +378,13 @@
 	for(var/i in 1 to 3)
 		new /obj/item/reagent_containers/pill/potassiodide(src)
 
-/obj/item/storage/pill_bottle/C2/probital
+/obj/item/storage/pill_bottle/probital

```

Comment:
Undocumented

Reply:
Undocumented but no unintentional. Change does not alter any behaviour.